cli/cmd/plugin/standalone-cluster/cmd/list.go

```diff
@@ -1,22 +1,26 @@
 // Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package main
+package cmd
 
 import (
 	"fmt"
-
```

Comment:
Blank line should stay to separate the groups of "standard lib", "third party lib", "local" imports.